WordPress/gutenberg
# Try possibly better method for Block Inserter Search focus

## Description


Try using useImperativeHandle for Block Inserter Search focus.

## How has this been tested?




Tested in Firefox with NVDA. Currently, changes only made to edit-post.

1. Open any Post or Page.
2. Select Block Inserter.
3. Focus should still be placed in Search input.

## Screenshots 

## Types of changes





Code enhancement

## Checklist:
- [x] I've updated related schemas if appropriate. <!-- Reference: https://github.com/WordPress/gutenberg/tree/trunk/schemas ->
